MeshLib/NodePartitionedMesh.h

```diff
@@ -29,6 +29,32 @@ namespace MeshLib
 class NodePartitionedMesh : public Mesh
 {
     public:
+        // Copy a global mesh for the case of the thread number is one,
+        // i.e the gobal mesh is not partitioned.
+        // \param mesh The gobal mesh
+        explicit NodePartitionedMesh(const Mesh& mesh)
+            : Mesh(mesh), _global_node_ids(mesh.getNumberOfNodes()),
+              _n_global_base_nodes(mesh.getNumberOfBaseNodes()),
+              _n_global_nodes(mesh.getNumberOfNodes()),
+              _n_active_base_nodes(mesh.getNumberOfBaseNodes()),
+              _n_active_nodes(mesh.getNumberOfNodes())
+        {
+            const auto& mesh_nodes = mesh.getNodes();
+            for (std::size_t i = 0; i < _nodes.size(); i++)
+            {
+                _global_node_ids[i] = _nodes[i]->getID();
+                auto node = _nodes[i];
+                // Copy constructor of Mesh does not copy the connected
+                // nodes to node.
```

Comment:
I am fine for this PR with the todo comment you added above. But maybe @rinkk or @endJunction can briefly tell why the Node constructor is incomplete?